src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/UpToDate/BuildUpToDateCheck.cs

```diff
@@ -110,8 +109,7 @@ protected override void Initialize()
 
         private void OnProjectChanged(IProjectSubscriptionUpdate e)
         {
-            var disableFastUpToDateCheckString = e.CurrentState.GetPropertyOrDefault(ConfigurationGeneral.SchemaName, ConfigurationGeneral.DisableFastUpToDateCheckProperty, null);
-            _isDisabled = disableFastUpToDateCheckString != null && string.Equals(disableFastUpToDateCheckString, TrueValue, StringComparison.OrdinalIgnoreCase);
+            _isDisabled = e.CurrentState.IsPropertyTrue(ConfigurationGeneral.SchemaName, ConfigurationGeneral.DisableFastUpToDateCheckProperty, false);
```

Comment:
Qualify `false` with the parameter name, that is, `defaultValue: false`.

Reply:
+1 on this `IsPropertyTrue` and `false` reads as odd without clarification

---
In reply to: [137062489](https://github.com/dotnet/project-system/pull/2785#discussion_r137062489) [](ancestors = 137062489)